src/apps/LandingRegister/LandingRegister.js

```diff
@@ -58,11 +57,26 @@ export default Radium(React.createClass({
   },
 
   handleButtonPress() {
+    this.refs.form.validateForm();
+  },
+
+  enableSubmit() {
     this.setState({
-      registerState: true
+      canSubmit: true
     });
   },
 
+  disableSubmit() {
+    this.setState({
+      canSubmit: false
+    });
+  },
+
+  submitForm(model) {
+    console.log('Model: ', model);
```

Comment:
console.log is temporary to show data access, will be replaced with calling Syncano webhook when all subpage styles are complete